spanner/google/cloud/spanner_v1/gapic/spanner_client.py

```diff
@@ -1449,9 +1459,9 @@ def begin_transaction(
     def commit(
         self,
         session,
-        mutations,
         transaction_id=None,
         single_use_transaction=None,
+        mutations=None,
```

Comment:
This should be treated as breaking. Existing users of the commit method will have code that fails/associates params with the incorrect data.